src/imf.jl

```diff
@@ -0,0 +1,78 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _imf{T<:AbstractFloat}(mass::AbstractVector{T}, expon::AbstractVector{T},
+                                mass_range::AbstractVector{T})
+    ne_comp = length(expon)
+    if length(mass_range) != ne_comp + 1
+        println("Length of array mass_range is not one more than that of expon")
```

Comment:
IDL AstroLib throws an error here, instead of returning a vector of zeros.  Why the different behavior?